backend/config/curation-api.yml

```diff
@@ -154,6 +154,8 @@ paths:
       description: >
         Fetch full metadata for the Collection and its Datasets.
 
+        If the collection owner's Authentication token is provided, then revision related fields will also be returned.
```

Comment:
1. Any user may request a public collection w/o authorization.
2. A curator may request a **private** collection that always requires authorization.